feat: update entity provider schedulers (#827)
janus-idp/backstage-plugins: plugins/aap-backend/README.md

1. Configure the scheduler using one of the following options:

>    - **Method 1**: If the scheduler is configured inside the `app-config.yaml` file, you can use it by adding the following code to the `packages/backend/src/plugins/catalog.ts` file:

Author: Same as [above](https://github.com/janus-idp/backstage-plugins/pull/827#discussion_r1352722633)

>    - **Method 1**: If the scheduler is configured inside the `app-config.yaml` file, you can use it by adding the following code to the `packages/backend/src/plugins/catalog.ts` file:

Reviewer: Please see my previous comment.

>    - **Method 1**: If the scheduler is configured inside the `app-config.yaml` via the schedule config key shown above, add the following code to `packages/backend/src/plugins/catalog.ts` file:

Reviewer: ```suggestion
   - **Method 1**: If the scheduler is configured inside the `app-config.yaml` using the schedule config key mentioned previously, add the following code to `packages/backend/src/plugins/catalog.ts` file:
```

     }
     ```

     ***

     **NOTE**

>      If any changes to the schedule in the `app-config.yaml` are made, a restart of the backend is necessary to apply the changes.

Reviewer: ```suggestion
     If you have made any changes to the schedule in the `app-config.yaml` file, then restart to apply the changes.
```

     ***

>    - **Method 2**: Add a schedule directly inside the `packages/backend/src/plugins/catalog.ts` file as follows:

Reviewer: ```suggestion
   - Add a schedule directly inside the `packages/backend/src/plugins/catalog.ts` file as follows:
```

Author: Same as [above](https://github.com/janus-idp/backstage-plugins/pull/827#discussion_r1352722633)

     }
     ```

   - If both the `schedule` (hard-coded schedule) and `scheduler` (`app-config.yaml` schedule) option are provided in the `packages/backend/src/plugins/catalog.ts`, the `scheduler` option takes precedence.

>      - If the schedule inside the `app-config.yaml` is not configured, then the `schedule` option is used.

Reviewer: ```suggestion
     - If the schedule inside the `app-config.yaml` file is not configured, then the `schedule` option is used.
```
